libs/vm-modules/src/math/tensor/tensor_estimator.cpp

```diff
@@ -42,7 +42,11 @@ using SizeVector = ArrayType::SizeVector;
 
 TensorEstimator::TensorEstimator(VMTensor &tensor)
   : tensor_{tensor}
-{}
+{
```

Comment:
I think you should revert all of this.

If you don't like the repeated  code, feel free to add a helper function in an anonymous namespace for getting the padded_size and size since those lines are repeated a lot

Reply:
done